src/plugins/intel_gpu/tests/unit/test_cases/strided_slice_gpu_test.cpp

```diff
@@ -33,9 +33,9 @@ class strided_slice_gpu: public ::testing::Test {
                 9.0f, 10.0f, 11.0f, 12.0f, 13.0f, 14.0f, 15.8f
         });
 
-        set_values<int64_t>(begin, {0, 0, 0, 0});
```

Comment:
As far as I know, the shape of begin/end/strides should be equal to the input rank
So its better change `ov::PartialShape{ 3 }` -> `ov::PartialShape{ 4 }` and leave the same `set_values`